dev/clint/src/clint/rules/track_api_usage_outermost.py

```diff
@@ -0,0 +1,42 @@
+import ast
+
+from clint.resolver import Resolver
+from clint.rules.base import Rule
+
+
+class TrackApiUsageOutermost(Rule):
+    def _message(self) -> str:
+        return (
+            "The `@track_api_usage` decorator must be applied as the outermost decorator "
+            "(first in the decorator list)."
+        )
+
+    @staticmethod
+    def check(
+        node: ast.expr,
+        resolver: Resolver,
+        decorator_list: list[ast.expr],
+    ) -> bool:
```

Comment:
```suggestion
    ) -> Optional[ast.expr]:
```

can we return the `track_api_usage` node?